django_mock_queries/query.py

```diff
@@ -156,11 +157,14 @@ def aggregate(self, *args, **kwargs):
     def order_by(self, *fields):
         results = self.items
         for field in reversed(fields):
+            if fields == '?':
```

Comment:
There is a typo here and this test fails. This should be `field`.

Reply:
yes, it's a typo error.
fixed.